client/my-sites/site-settings/form-security.jsx

```diff
@@ -96,7 +97,7 @@ class SiteSettingsFormSecurity extends Component {
 
 const connectComponent = connect( ( state ) => {
 	const siteId = getSelectedSiteId( state );
-	const isAtomic = isSiteAutomatedTransfer( siteId );
+	const siteIsAutomatedTransfer = isSiteAutomatedTransfer( state, siteId );
```

Comment:
We have a lot of instances of `isAtomic` throughout the repo - most stemming from a call to `isSiteAutomatedTransfer`, and a couple that derive this from site options. If we feel this is something that needs a refactor, then might be best to do it holistically maybe. Not sure if it's the right decision though:

1. The selector could have been named anything. The inference from it seems to be the "atomic" status of a site.
2. I feel `isAtomic` may have evolved into a bit of a de facto convention, as `siteIsAutomatedTransfer` only appears in these two other files modified 4 years ago.

Naming is hard! What is more acceptable: "atomic site" or "automated transfer site"?

Reply:
Ahh haha those were the two files that I was working on it seems. I got this backward. I will undo that piece.